### ISSUE TITLE
Getting error  react native 0.68 on android (Plugin with id ‘maven’ not found. )

### ISSUE THREAD

**Issue by bfk1690** [sijav/react-native-ip-sec-vpn#35]
I am getting error when buil project on android.
build.gradle file error Plugin with id ‘maven’ not found.

Please help me.

**Comment by jhontech32** [comment]
face the same issues here

**Comment by EGENIUS2** [comment]
Same issues here

**Comment by clement66000** [comment]
Same issue here

**Comment by AftabUfaq** [comment]
any solution

**Comment by liaostr** [comment]
Same issue here

**Comment by Rais30** [comment]
face the same issues here

**Comment by hkhawar21** [comment]
I am facing the same issue as well
I am using Expo Managed Workflow for my app

Expo SDK: 46
React Native Version: 0.69.6

**Comment by MrAbhishekSingh** [comment]
![Screenshot from 2022-12-07 19-58-34](https://user-images.githubusercontent.com/101648029/206205946-2f55e50b-da9c-4161-8b86-77ada4f45c94.png)

**Comment by MrAbhishekSingh** [comment]
> ![Screenshot from 2022-12-07 19-58-34](https://user-images.githubusercontent.com/101648029/206205946-2f55e50b-da9c-4161-8b86-77ada4f45c94.png)

go to nodemodule and change maven to maven-publish

**Comment by bfk1690** [comment]
> > ![Screenshot from 2022-12-07 19-58-34](https://user-images.githubusercontent.com/101648029/206205946-2f55e50b-da9c-4161-8b86-77ada4f45c94.png)
> 
> go to nodemodule and change maven to maven-publish

Has anyone tried this solution?

**Comment by MrAbhishekSingh** [comment]
Yes go to you node module and search same pack name and go to build.gradle and search maven and change maven to maven-publish

**Comment by hkhawar21** [comment]
@MrAbhishekSingh are you using expo managed workflow?

**Comment by MrAbhishekSingh** [comment]
> @MrAbhishekSingh are you using expo managed workflow?

No,it's react native cli

**Comment by fajrur** [comment]
> > ![Screenshot from 2022-12-07 19-58-34](https://user-images.githubusercontent.com/101648029/206205946-2f55e50b-da9c-4161-8b86-77ada4f45c94.png)
> 
> go to nodemodule and change maven to maven-publish

Thank you, it's worked for me

**Comment by Bimbimz** [comment]
node_modules\react-native-ip-sec-vpn\android\build.gradle' line: 152
Configuration with name 'compile' not found.

how to fix this?

**Comment by Bimbimz** [comment]
what version of react are you using?
iam try 0.68.2 and 0.69.6 but not work.
when i change `maven` to `maven-publish` it just creates a new problem:
- Configuration with name 'compile' not found.
- Could not find mavenDeployer() for arguments.

**Comment by Bimbimz** [comment]
> ![Screenshot from 2022-12-07 19-58-34](https://user-images.githubusercontent.com/101648029/206205946-2f55e50b-da9c-4161-8b86-77ada4f45c94.png)

what version of react are you using?
iam try 0.68.2 and 0.69.6 but not work.
when i change `maven` to `maven-publish` it just creates a new problem:
- Configuration with name 'compile' not found.
- Could not find mavenDeployer() for arguments.

**Comment by AbhishekSinghT** [comment]
Can you share me the actual problem with screen shot

**Comment by Bimbimz** [comment]
> Can you share me the actual problem with screen shot

![Screenshot 55](https://user-images.githubusercontent.com/59421003/209912569-6142b808-0ddd-4d0a-b925-e0ffc266109c.png)

Configuration with name 'compile' not found.

**Comment by AbhishekSinghT** [comment]
Just go to down in same page and comment out the code ..you can take ref from this image

**Comment by AbhishekSinghT** [comment]
If you are developing VPN app so you can use this library:-. https://www.npmjs.com/package/react-native-simple-openvpn

**Comment by Bimbimz** [comment]
Can you explain in more detail which lines should I comment out?
I see your image is broken.

i had success with react-native-simple-openvpn.
but somehow its not working after release on playstore.

**Comment by AbhishekSinghT** [comment]
You have some line added in your project then you can build. Your app its also work on in playstore.
You can read this page and add that line in your project it's will help for you..
https://stackoverflow.com/questions/71206844/vpn-is-not-connecting-after-uploading-android-app-bundle-aab-to-play-store


Some day ago I'm also faced same problem in my app not working after deployed on playstore 
But now it's working:- https://play.google.com/store/apps/details?id=com.uaevpn

**Comment by Bimbimz** [comment]
Wow shocking news.
I deleted the project and don't have a backup after trying to repair and upload it up to 3x to playstore. it took about 7days to see results from playstore with results they ask for a video how to run it. maybe that's why they suspect that my app is spam.

**Comment by Bimbimz** [comment]
yes i also see react-native-ipsec-vpn gives the same argument.
`android.bundle.enableUncompressedNativeLibs`

**Comment by Bimbimz** [comment]
release thx
https://play.google.com/store/apps/details?id=com.bimlind.vpn

**Comment by Bimbimz** [comment]
> You have some line added in your project then you can build. Your app its also work on in playstore. You can read this page and add that line in your project it's will help for you.. https://stackoverflow.com/questions/71206844/vpn-is-not-connecting-after-uploading-android-app-bundle-aab-to-play-store
> 
> Some day ago I'm also faced same problem in my app not working after deployed on playstore But now it's working:- https://play.google.com/store/apps/details?id=com.uaevpn

release thx
https://play.google.com/store/apps/details?id=com.bimlind.vpn

**Comment by zoobibackups** [comment]
I have new package for this please use this. best for both android and iOS . 
https://github.com/shadow-boy/react-native-vpn-ipsec

**Comment by AbhishekSinghT** [comment]
user authentication failed in release mode, but working in debugging mode fine,  https://github.com/shadow-boy/react-native-vpn-ipsec

**Comment by AbhishekSinghT** [comment]
> I have new package for this please use this. best for both android and iOS . https://github.com/shadow-boy/react-native-vpn-ipsec

user authentication failed in this package any solution?
